treepoem/__init__.py

```diff
@@ -57,7 +58,31 @@
 
 """ + BASE_PS
 
-BBOX_COMMAND = ['gs', '-sDEVICE=bbox', '-dBATCH', '-dSAFER', '-']
+def _gs_command():
+    if sys.platform != 'win32':
+        return 'gs'
+
+    # platform is Windows
+    try:
+        # has executable been defined by 'gssetgs.bat'?
+        cmd = os.environ['GSC']
+    except KeyError:
+        pass
+    else:
+        return cmd
+
+    cmd = 'gswin64c'
+    try:
+        subprocess.check_call([cmd, '-dBATCH'])
+    except Exception:
```

Comment:
`except Exception` is an anti-pattern, please catch only the exception type(s) that are possible